static/espeasy.js

```diff
@@ -79,7 +79,7 @@ var commonPlugins = [
   //P124
   "multirelay", "multirelay,on", "multirelay,off", "multirelay,set", "multirelay,get", "multirelay,loop",
   //P126
-  "ShiftOut", "ShiftOut,Set", "ShiftOut,SetNoUpdate", "ShiftOut,Update", "ShiftOut,SetAll", "ShiftOut,SetAllNoUpdate", "ShiftOut,SetAllLow", "ShiftOut,SetChipCount", "ShiftOut,SetHexBin",
+  "ShiftOut", "ShiftOut,Set", "ShiftOut,SetNoUpdate", "ShiftOut,Update", "ShiftOut,SetAll", "ShiftOut,SetAllNoUpdate", "ShiftOut,SetAllLow", "ShiftOut,SetAllHigh", "ShiftOut,SetChipCount", "ShiftOut,SetHexBin",
```

Comment:
I'd expect some of the [P129] [ShiftIn commands](https://espeasy.readthedocs.io/en/latest/Plugin/P129.html#commands) here as well?

Reply:
And you shall have them.... 🙂

Reply:
It's actually P129, but at least they're in now 😜 

Once this is merged developers should update this list/file when adding commands, maybe you could add that as a 'task' to the developer documentation?

Reply:
i don’t know what happend...the 9 went upside down....

> maybe you could add that as a 'task' to the developer documentation?

where do i find this?

Reply:
> where do i find this?

Ii think it should be part of this chapter: https://github.com/letscontrolit/ESPEasy/blob/mega/docs/source/Participate/PlatformIO.rst#writing-documentation
Or add an extra paragraph for explaining how to update these files.